api/proto/teleport/accesslist/v1/accesslist_service.proto

```diff
@@ -53,6 +55,8 @@ service AccessListService {
   rpc GetAccessListMember(GetAccessListMemberRequest) returns (Member);
   // UpsertAccessListMember creates or updates an access list member resource.
   rpc UpsertAccessListMember(UpsertAccessListMemberRequest) returns (Member);
+  // UpdateAccessListMember conditionally updates an access list member resource.
```

Comment:
What are the "expected conditions"? I don't see any field in the request that allows you to specify a condition.
If the update is not guaranteed, how can we tell you if anything has been updated? Do you need to compare the returned member with the expected one?

Reply:
It's the revision and lives in metadata.revision